examples/kubernetes/prometheus-k8s-service-monitor-cilium.yaml

```diff
@@ -0,0 +1,21 @@
+---
+apiVersion: monitoring.coreos.com/v1
+kind: ServiceMonitor
+metadata:
+  name: cilium-metrics
+  labels:
+    k8s-app: cilium-metrics
+  namespace: monitoring
+spec:
+  jobLabel: k8s-app
+  selector:
+    matchLabels:
+      k8s-app: cilium-metrics-svc
+      app: cilium-metrics-svc
```

Comment:
I'm not sure this will work as intended. The service backs to cilium-agent pods but would get a different one depending on the DNS return for that service (or perhaps it always goes to the most local service backend?) In any case, if the intention is to scrape all cilium-agent pods, selecting those would present the collector with all the right IPs to connect to (and the port, since that is annotated on each pod along with a `scrape: true` flag).